packages/slate-editor/src/components/EditorBlock/getMenuPopperModifiers.ts

```diff
@@ -0,0 +1,115 @@
+import type { Rect } from '@popperjs/core';
+import type { Modifier } from 'react-popper';
+
+import type { PopperOptionsContextType } from '#modules/popper-options-context';
+
+import styles from './Menu.module.scss';
+
+/**
+ * Elements in slate are surrounded by an outline,
+ * which is not included in the element's bounding box.
+ */
+const TETHER_OFFSET_OUTLINE_SIZE = 6;
+
+/**
+ * Since slate element has outline on top and bottom,
+ * we need to multiply the outline size by 2 to get the total offset.
+ */
+const SLATE_ELEMENT_TOTAL_OFFSET = 2 * TETHER_OFFSET_OUTLINE_SIZE;
+
+export function getMenuPopperModifiers(
+    popperOptions: PopperOptionsContextType,
+): Modifier<string>[] {
+    const { modifiers } = popperOptions;
+
+    return [
+        {
+            name: 'offset',
+            enabled: true,
+            options: {
+                offset: ({
+                    popper,
+                    reference,
+                }: {
+                    popper: Rect;
+                    reference: Rect;
+                }): [number, number] => {
+                    const referenceHeight = getSlateElementHeight(reference.height);
+                    const popperTallerThanReference = popper.height - referenceHeight;
```

Comment:
I think it's better to call this variable `popperHeightDifference`. Current naming implies that this is a boolean.

Reply:
yeah, i tried to also some logic inside the name, just to indicate what i expect from this variable